### ISSUE TITLE
Cannot compile. Module parse failed: Unexpected token (470:30)

### ISSUE THREAD

**Issue by aljohn-ma** [icehaunter/vue3-datepicker#19]
Failed to compile.

./node_modules/vue3-datepicker/dist/vue3-datepicker.esm.js 470:30
Module parse failed: Unexpected token (470:30)
You may need an appropriate loader to handle this file type.
|         });
|         const isEnabled = (target, lower, upper, disabledDates) => {
>             if (disabledDates?.dates?.some(date => isSameDay(target, date)))
|                 return false;
|             if (!lower && !upper)

**Comment by icehaunter** [comment]
Oh, good catch, I'll fix it ASAP

**Comment by aljohn-ma** [comment]
> Oh, good catch, I'll fix it ASAP

Nice!!! Thanks. I'll be waiting for the fix.

**Comment by pounard** [comment]
Any news on this ? I get the same error, sadly.

**Comment by pounard** [comment]
For information, I tried replacing the import using `import Datepicker from 'vue3-datepicker/src/datepicker/Datepicker' and got this error:

```
ERROR in ./node_modules/vue3-datepicker/src/datepicker/Datepicker.vue?vue&type=script&lang=ts (./node_modules/vue-loader/dist??ref--5-0!./node_modules/vue3-datepicker/src/datepicker/Datepicker.vue?vue&type=script&lang=ts) 23:17
Module parse failed: Unexpected token (23:17)
File was processed with these loaders:
 * ./node_modules/vue-loader/dist/index.js
You may need an additional loader to handle the result of these loaders.
|      */
|     modelValue: {
>       type: Date as PropType<Date>,
|       required: false,
|     },
 @ ./node_modules/vue3-datepicker/src/datepicker/Datepicker.vue?vue&type=script&lang=ts 1:0-113 1:0-113 1:114-216 1:114-216
 @ ./node_modules/vue3-datepicker/src/datepicker/Datepicker.vue
 @ ./node_modules/babel-loader/lib??ref--6-0!./node_modules/ts-loader??ref--6-1!./node_modules/vue-loader/dist??ref--5-0!./assets/js/Shared/common/DatepickerApp.vue?vue&type=script&lang=ts
 @ ./assets/js/Shared/common/DatepickerApp.vue?vue&type=script&lang=ts
 @ ./assets/js/Shared/common/DatepickerApp.vue
 @ ./assets/js/Shared/common/datepicker.ts
 @ ./assets/js/app.gestion.ts
```

**Comment by icehaunter** [comment]
Yeah, sorry, didn't have the time. I should be able to fix it within 24h from now. I'll reply here when it's done

**Comment by icehaunter** [comment]
Should be fixed and released as v0.2.3!

I've set TS compilation options to ES6. If someone thinks that's unreasonable, please voice your concerns. But if your requirements are more specific, you should be able to import the `.vue` component directly and transpile it as you need.

And sorry for the delay everyone, real life got in the way, as always :smile:

**Comment by pounard** [comment]
Thank you very much !
